source/wp-content/themes/wporg-news-2021/block-templates/category-month-in-wordpress.html

```diff
@@ -0,0 +1,16 @@
+<!-- wp:template-part {"slug":"header","align":"full","className":"site-header-container"} /-->
+<!-- This is a hack to ensure content isn't covered by the fixed header -->
+<!-- wp:template-part {"slug":"header","align":"full","className":"site-header-offset"} /-->
+
+<!-- wp:query {"tagName":"main","className":"site-content-container","displayLayout":{"type":"flex","columns":4},"query":{"perPage":"10000","inherit":false}} -->
```

Comment:
I'm not sure of a good way to put the blue paintbrush strokes in-between the years 🤔 

We can't group the posts into years and then loop through each group, since Gutenberg doesn't support dynamic templates - https://github.com/WordPress/gutenberg/issues/32939

_Aside: That one issue is responsible for 70% of the difficulties I've run into :/ . I've been tracking them at https://github.com/WordPress/wporg-news-2021/issues?q=is%3Aissue+is%3Aopen+label%3A%22blocked%3A+external%22, so it'd be good to add this one as well if we don't figure out a solution._

We could maybe try something with CSS, like

```css
body.category.month-in-wordpress .entry-header nth-of-type(12)::after {
    background-image: url(...)
}
```

...but that would need to account for how the current won't have 12 months, and previous years won't always have 12 months.

Reply:
At `1300px` some of the columns jump off the screen. 

I'm guessing some of the CSS alignment that's applied to typical categories is causing that. some of the containers might need `alignwide` to avoid that.

you might also need to change some of the alignment/post-content styles to apply to `body.category:not(.month-in-wordpress)` or something. 

I'm happy to dig in more if you run into any questions about how it's currently setup or anything.

Reply:
I think the query will also need `categoryIDs` to limit it to just the MIW cat.

Reply:
_( er, I meant these to be 3 separate threads, but all related to the query block. I guess GH combines them if they're for the same line :( )_

Reply:
After tinkering a bit, the best idea I have for the spacer between years is with a PHP filter that hooks `post_class`, and adds conditional classes to the post for various special contexts that we need. For instance a `first-in-year` or `last-in-year` class. Then use `::after` to deal with it in CSS.

A similar hack might work for other situations where conditional formatting is needed but the logic is too complex for CSS. Like the conditional blue backgrounds for #76.

It's not a great solution but would move things along. Do you see any problems with that?

Reply:
Yeah, that that sounds like a good workaround 👍🏻